CMakeLists.txt

```diff
@@ -653,10 +653,12 @@ function(get_link_libraries OUT TARGET)
     set(${OUT} ${RESULT} PARENT_SCOPE)
 endfunction()
 
-get_link_libraries(PRIVATE_LIBS vncserver)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/libvncserver.pc.cmakein ${CMAKE_CURRENT_BINARY_DIR}/libvncserver.pc @ONLY)
-get_link_libraries(PRIVATE_LIBS vncclient)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/libvncclient.pc.cmakein ${CMAKE_CURRENT_BINARY_DIR}/libvncclient.pc @ONLY)
+if(NOT WIN32)
```

Comment:
why not?

Reply:
There is no pkg-config on Windows.

Reply:
Well, not per default, but there might be: https://www.freedesktop.org/wiki/Software/pkg-config/

Reply:
Well, then let's keep it. A cmake config file would be a better thing to pick up.